selenium_tests/lti_emailer/page_objects/emailer_mainpage.py

```diff
@@ -0,0 +1,23 @@
+from selenium.webdriver.common.by import By
+from selenium_tests.lti_emailer.page_objects.emailer_base import EmailerBasePage
+
+# These are page-specific locators.
+class Locator1(object):
+      _h1_selector = (By.CSS_SELECTOR, "h1")
+      _emailer_list = (By.ID, "email-section-1670")
+
+
+
+class MainPage(EmailerBasePage):
+
+    def get_title(self):
+        element = self.find_element(*Locator1._h1_selector)
+        return element
+
+    def get_addy(self):
```

Comment:
I'm not sure what this does... it isn't used anywhere, and the code is a bit ambiguous. Perhaps a clearer method name and some comments would help (why are we looking at `_emailer_list`? Why do we return `.text.`?).